src/System.Reflection.Emit/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,13 @@
+Compat issues with assembly System.Reflection.Emit:
+TypesMustExist : Type 'System.Reflection.Emit.AssemblyBuilder' does not exist in the implementation but it does exist in the contract.
```

Comment:
What configuration is this baseline file needed for?

Reply:
it was for netstandard, sorry I didn't add it to the file name, I'll fix that.